.gitignore

```diff
@@ -10,6 +10,7 @@
 
 # production
 /build
+.idea
```

Comment:
What does this line means, broo

Reply:
This is a folder was auto created by PHPStorm and we need to ignore it

Reply:
OK bro